packages/devtools_app/lib/src/ui/service_extension_widgets.dart

```diff
@@ -421,17 +421,19 @@ mixin _ServiceExtensionMixin<T extends _ServiceExtensionWidget> on State<T> {
     try {
       await action();
 
-      if (widget.completedText != null) {
+      if (mounted && widget.completedText != null) {
         Notifications.of(context).push(widget.completedText);
       }
     } catch (e, st) {
       log('$e\n$st');
 
       Notifications.of(context).push(widget.describeError(e));
```

Comment:
shouldn't we also short circuit displaying the notification if the widget is not mounted?

Reply:
done